lib/stemcell/option_parser.rb

```diff
@@ -364,6 +383,13 @@ def parse!(args)
       # convert ephemeral_devices from comma separated string to ruby array
       options['ephemeral_devices'] &&= options['ephemeral_devices'].split(',')
 
+      # format the classic link options
+      options['classic_link'] ||= {}
+      options['classic_link']['vpc_id'] = \
+        options['classic_link_vpc_id'] if options['classic_link_vpc_id']
+      options['classic_link']['security_group_ids'] = \
+        options['classic_link_security_group_ids'] if options['classic_link_security_group_ids']
```

Comment:
Maybe put the `if` statements on their own line and put the assignments in the block?